api-reference/10 UI Components/dxDropDownList/1 Configuration/useItemTextAsTitle.md

```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether the **{WidgetName}** uses item's text a **title** attribute.
```

Comment:
Let's add a link to title:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title